docs/tutorials/sampling/ResamplingTutorial.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+===================================
+Particle Filter Resamplers: Example
```

Comment:
```suggestion
Particle Filter Resamplers
```
Is this a tutorial or an example? Getting philosophical here, but I'm wondering how we define the difference, so I understand your confusion. Either way, make up your mind! 

To be honest, tutorial/example titling could do with more consistency across the board. Cohesiveness on the documentation will inspire confidence in the software. I'll have a look at this at some point soon.